sdk/schemaregistry/azure-schemaregistry/swagger/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Schema Registry for Python
+
+> see https://aka.ms/autorest
+
+### Setup
+```ps
+cd C:\work
+git clone --recursive https://github.com/Azure/autorest.python.git
+cd autorest.python
+git checkout azure-core
+npm install
+```
+### Generation
+```ps
+cd C:\Work\SchemaRegistry\
+autorest --use=@autorest/python@5.0.0-preview.6
+```
+### Settings
+``` yaml
+input-file: https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/schemaregistry/azure-data-schemaregistry/swagger/swagger.json
```

Comment:
Let's make sure we use a Swagger from the spec repo eventually

Reply:
Updated to use https://raw.githubusercontent.com/Azure/azure-rest-api-specs/arerlend.sr.init/specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json.
Will update again once it's merged to master.